.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
```

Comment:
The only danger I see is that if someone might be put off by a wall of text and not submit a report, but that's probably not a common case. Since the purpose of this is to get as much info upfront as we can, makes sense to me!